adapter/repository/in-memory-simple/src/test/java/org/mobilitydata/gtfsvalidator/db/InMemoryGtfsDataRepositoryTest.java

```diff
@@ -373,6 +373,40 @@ void addTransferAndGetTransferByStopIdPairShouldReturnSameEntity() {
         assertEquals(mockTransfer03, underTest.getTransferByStopPair("stop id 0", "stop id 5"));
     }
 
+    @Test
+    void getTransferAllShouldReturnTransferCollection() {
```

Comment:
Shall we check for immutability as well (invoking `put` should raise an exception)

Reply:
Will report this verification for all gtfs entities